add photos gallery 1x 2x

vadimvitamin opened the pull request:
mobile , tablet ,desktop

vadimvitamin committed:
add photos gallery 1x 2x

mobile , tablet ,desktop

Oleksandr-Pavlov commented on src/sass/layot/_galery.scss:
```diff
@@ -0,0 +1,22 @@
+.gallery {
+    background-color: color#fff5f6;
+    width: 320px;
```
Навіщо задавати ширину? Все, що менше 480рх має бути респонсивним. Тут не треба задавати ширину взагалі.

Oleksandr-Pavlov commented on src/sass/layot/_galery.scss:
```diff
@@ -0,0 +1,22 @@
+    width: 320px;
+ }
+ 
+ .container {
```
Цей селектор не потрібен. Він вже є в базових стилях https://prnt.sc/6VHRF7kgWX5P

Oleksandr-Pavlov commented on src/sass/layot/_galery.scss:
```diff
@@ -0,0 +1,22 @@
+     margin-left: auto;
+     margin-right: auto;
+ }
+ .gallery__list {
```
list-style: none;
margin-left: auto;
margin-right: auto;

Це не потрібно. Воно вже є в базових стилях скидання

vadimvitamin replied:
це було так до того як були створені базові стилі)))

Oleksandr-Pavlov commented on src/sass/layot/_galery.scss:
```diff
@@ -0,0 +1,22 @@
+     display: grid;
+     grid-template-columns: repeat(6, 1fl);
+     grid-template-rows:45px;
+     @media screen and (min-width:320px) {
```
Навіщо тут медіа запит на 320+рх? Знову ж таки, від 480 і менше у нас респонсивна верстка

vadimvitamin replied:
цце все я вже забрав.

Oleksandr-Pavlov reviewed (changes requested):
Прошу внести правки по коментарях. Плюс, зверни увагу, що ти, схоже, скачав не останню версію репозиторію, тому що помилки в назвах папок (як тут https://prnt.sc/G7lEtgGMx6hg) я вже виправляв. Також не побачив адаптацію під планшет і декстоп. І не побачив твою частину html.

vadimvitamin committed:
add photos gallery 1x 2x

mobile , tablet ,desktop
зробив розмітку , сспробував розташувати елементи з допомогою grid

vadimvitamin committed:
9.44
